src/app/components/LinkedData/index.jsx

```diff
@@ -104,6 +107,9 @@ LinkedData.propTypes = {
 
 LinkedData.defaultProps = {
   about: undefined,
+
+  // custom propType checks used
```

Comment:
I didn't want to disable the eslint error for the entire page, because it's a good reminder to set a default, but using a `eslint-disable-next-line` multiple times really cluttered up the file. 

So I prefer the comment 🤷‍♀